package.json

```diff
@@ -4,6 +4,8 @@
     "scripts": {
         "lint": "eslint --ext .js,.vue resources/js",
         "lint:fix": "eslint --fix --ext .js,.vue resources/js",
+        "stylelint": "stylelint '**/*.scss' '**/*.vue'",
```

Comment:
As mentioned in the workflow file: we can add the `lint:` prefix to to anything ESlint/Stylelint related and then have the `lint` script run it all. The same goes for fix.

Reply:
Done!